src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -1,35 +1,68 @@
 package com.kodilla.ecommercee.controller;
-
+import com.kodilla.ecommercee.domain.OrderItem;
 import com.kodilla.ecommercee.domain.dto.CartDto;
 import com.kodilla.ecommercee.domain.dto.ProductDto;
+import com.kodilla.ecommercee.exception.NotFoundException;
+import com.kodilla.ecommercee.mapper.CartMapper;
+import com.kodilla.ecommercee.mapper.ProductMapper;
+import com.kodilla.ecommercee.service.CartService;
+import com.kodilla.ecommercee.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.MediaType;
+
 import org.springframework.web.bind.annotation.*;
 
 import java.util.ArrayList;
 import java.util.List;
 
+@CrossOrigin(origins = "*")
 @RestController
-@RequestMapping("api/v1/ecommercee/carts")
+@RequestMapping("api/v1/ecommercee/cart")
```

Comment:
stosujemy liczbe mnogą tak jak w podanym przeze mnie wcześniej standardzie
```suggestion
@RequestMapping("api/v1/ecommercee/carts")
```